src/lm.jl

```diff
@@ -185,6 +185,31 @@ function adjr2(obj::LinearModel)
     1 - (1 - r²(obj))*(n-1)/(n-p)
 end
 
+struct FStatistic
+    nobs::Float64
+    ndims::Int64
+    f::Float64
+    fprob::Float64
+end
```

Comment:
If you expect this to always be the return type of `fstatistic`, then it should be included in `StatsModels` with the function stub.

Reply:
Thanks, I'll add that.